tfe/resource_tfe_policy.go

```diff
@@ -0,0 +1,308 @@
+package tfe
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/go-tfe"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func resourceTFEPolicy() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceTFEPolicyCreate,
+		Read:   resourceTFEPolicyRead,
+		Update: resourceTFEPolicyUpdate,
+		Delete: resourceTFEPolicyDelete,
+
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceTFEPolicyImporter,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Description: "The name of the policy",
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+			},
+
+			"description": {
+				Description: "Text describing the policy's purpose",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+
+			"organization": {
+				Description: "Name of the organization that this policy belongs to",
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+			},
+
+			"kind": {
```

Comment:
My guess is that changing this argument should be a ForceNew as well. I don't think it is updatable?